diff --git a/inputs/pqshield-signature-zoo-20260928/sources/sqisign/src/quaternion/ref/include/quaternion.h b/inputs/pqshield-signature-zoo-20260928/sources/sqisign/src/quaternion/ref/include/quaternion.h
index 34a098ca89..911ac6f59b 100644
--- a/inputs/pqshield-signature-zoo-20260928/sources/sqisign/src/quaternion/ref/include/quaternion.h
+++ b/inputs/pqshield-signature-zoo-20260928/sources/sqisign/src/quaternion/ref/include/quaternion.h
@@ -433,13 +433,9 @@ int quat_ideal_small_equivalent_coprime(quat_alg_elem_t *gen,
  */
 
 /**
-<<<<<<< HEAD
  * @brief QuaternionResponseComputation: Function to compute a response ideal
  *
  * Computes sk_chall_ideal and and resp_quat, the response ideal's norm and sk_chall_quat to obtain sk_chall_ideal
-=======
- * @brief QuaternionResponseComputation: Function to compute an response ideal resp
->>>>>>> main
  *
  * @param sk_chall_ideal Output:  ideal of small prime norm equivalent to the intersection of skideal with
  * ideal_chall_two
